watertap/property_models/ion_DSPMDE_prop_pack.py

```diff
@@ -252,6 +256,15 @@ def build(self):
             units=pyunits.Pa * pyunits.s,
             doc="Fluid viscosity",
         )
+        # Ion electrical mobility
+        self.electrical_mobility_comp = Param(
+            self.ion_set | self.solute_set,
+            mutable=True,
+            default=5.19e-8,  # default as Na+
+            initialize=self.config.electrical_mobility_data,
+            units=pyunits.meter**2 * pyunits.volt**-1 * pyunits.second**-1,
+            doc="Ion electrical mobility",
+        )
```

Comment:
This parameter is indexed by `ion_set` and `solute_set` (meaning that it is currently indexed for all dissolved species (both charged and uncharged). However, when later used to calculate conductivity you only iterate over and use these values from the `ion_set`.

Should this parameter only be for the `ion_set` or should the calculation of conductivity iterate over both the `ion_set` and `solute_set`? (Or does it really matter?)

Reply:
Per conversation with Xiangyu, this indexing is necessary because of how the solutes (both charged and uncharged) are specified in the configuration args.

Reply:
I changed the index here to be `ion_set | solute_set`.  Per discussion with Austin, we may modify the `solute_list` configuration in the future to make charge and electrical mobility indexed by `ion_set` only.  For now the two were indexed also by neutral species whose values will be set zero. No issues by this way for the current unit models.